src/dapp/README.md

```diff
@@ -11,6 +11,10 @@ Please make sure you have [`solc` installed in your CLI](https://solidity.readth
    make uninstall             uninstall dapp(1) from /usr/local
 ```
 
+### macOS
+
+On macOS, you need to install Bash 4.
```

Comment:
But only if they aren't installing via nix.

Reply:
Isn’t that clear from context?  All of the section is tailored at non-nix users, so I’m wondering if we should add a separate nix one.